src/scribe_data/checkquery.py

```diff
@@ -40,11 +40,19 @@ class QueryFile:
     def load(self, limit: int) -> str:
         """Load the SPARQL query from 'path' into a string.
 
-        Args:
-            limit (int): the maximum number of results a query should return.
+        Parameters
+    -----------
+        source_value : str
+            The source value to find equivalents for (e.g. 'english').
+        source_key : str
+            The source key to reference (e.g. 'language').
+        target_key : str
+            The key to target (e.g. 'iso').
+        error_msg : str
+            The message displayed when a value cannot be found.
 
         Returns:
-            str: the SPARQL query.
+            The 'target' value given the passed arguments.
```

Comment:
We want this here to be the same as it was before, but with a different format for the `Returns:`. So ideally:

```
Returns
--------
    str : the SPARQL query.
```